utils/templates/module-member-absent.yml.in

```diff
@@ -0,0 +1,12 @@
+---
+- name: ${name^} absent test
+  hosts: ipaserver
+  become: true
+
+  tasks:
+  - name: Ensure $name NAME is absent
+    ipa$name:
+     name: NAME
```

Comment:
The `standard password` should be added here: `ipaadmin_password: SomeADMINpassword`.

Also, it seems only a single space is used to ident the block, and we use two.